README.md

```diff
@@ -68,7 +68,7 @@ Exp2    ::=  Exp3 (relop Exp3)?
 Exp3    ::=  Exp4 (addop Exp4)*
 Exp4    ::=  Exp5 (mulop Exp5)*
 Exp5    ::=  prefixop? Exp6
-Exp6    ::=  (Exp7 unary?) | (unary Exp7)
+Exp6    ::=  (Exp7 '**')* Exp7
```

Comment:
We haven't solved our problem with operator precedence - this should be `Exp6 ::= Exp7 ('**' Exp5)?